File: PHP/car.php
<?php

class Car
{
    public $id;
    public $license;
    public $driver;
    public $passengers;

    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar()
    {
        echo "
            <br/>
            Licencia: $this->license  <br/>
            Driver: {$this->driver->name} <br/>
            Número de pasajeros: $this->passengers <br/>
        ";
    }

    public function getPassenger()
    {
        return $this->passengers;
    }

    public function setPassenger($passengers)
    {

        if ($passengers == 4) {
            $this->passengers = $passengers;
        } else {
            echo "<br/> Error: Necesitas asignar 4 pasajeros. <br/>";
        }
    }
}

File: Java/Car.java
package Java;

class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;

    }

    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger(Integer passenger) {
        if (passenger == 4) {
            this.passenger = passenger;
        } else {
            System.out.println("Error: UberX soporta sólamente 4 pasajeros.");
        }
    }

    void printDataCar() {
        if (passenger != null) {
            System.out.println("License: " + license + ", Driver name: " + driver.name + ", passengers: " + passenger);
        }
    }
}